otelcollector/deploy/addon-chart/azure-monitor-metrics-addon/templates/ama-metrics-daemonset.yaml

```diff
@@ -299,11 +299,11 @@ spec:
           imagePullPolicy: IfNotPresent
           resources:
             limits:
-              cpu: 200m
+              cpu: 500m
               memory: 1Gi
             requests:
-              cpu: 75m
-              memory: 150Mi
+              cpu: 500m
+              memory: 1Gi
```

Comment:
kaveesh - previously request wasn’t equal to limits and that’s another change as part of this pr as well?

Reply:
Yeah, I noticed that we didn't have requests == limits over here and its like this in the AKS-RP too. I'm going to be updating both (prom collector and AKS-RP) to the same values with this change